core/templates/dev/head/components/question-directives/question-player/review-and-retry-modal.template.html

```diff
@@ -0,0 +1,17 @@
+<div class="modal-header">
+  <h3> <[currentSkill]> </h3>
+</div>
+
+<div class="modal-body">
+  <concept-card skill-ids="failedSkillIds" index="index"> </concept-card>
+</div>
+
+<div class="modal-footer">
+  <button class="btn btn-default" ng-click="closeModal()"> Cancel </button>
+  <button class="btn btn-success" ng-if="!isLastConceptCard()" ng-click="goToNextConceptCard()"> Next </button>
+  <button class="btn btn-success"
+          ng-if="isLastConceptCard()"
```

Comment:
Can we keep the format same, the above line is all in the same line, but here you have it line by line. Also, why does the retry test button have btn-success (this would color it green right?)..

Reply:
Oh I just realized that only one of Next or retry test would be visible at any point. In that case I think the color is ok.